library/ssl_tls13_generic.c

```diff
@@ -435,41 +439,40 @@ int mbedtls_ssl_parse_signature_algorithms_ext( mbedtls_ssl_context *ssl,
 
```

Comment:
Duplicated extensions are not permitted in TLS 1.3, so it shouldn't happen that `received_signature_schemes_list` is not `NULL`. I'd suggest to either remove this check or turn it into an assertion.

Reply:
It's a separate issue to check the state of the rejection of duplicated extensions.